apps/server/src/modules/learnroom/controller/api-test/dashboard.api.spec.ts

```diff
@@ -134,7 +134,7 @@ describe('Dashboard Controller (API)', () => {
 			expect(response.status).toEqual(200);
 			const body = response.body as DashboardResponse;
 			expect(body.id).toEqual(dashboardId);
```

Comment:
The test `it('should return dashboard with teacher and substitute teacher active courses', ...` must be removed.

Reply:
removed